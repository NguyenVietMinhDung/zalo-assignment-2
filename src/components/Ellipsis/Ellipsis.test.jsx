import React from 'react';
import { shallow } from 'enzyme';
import Ellipsis from './Ellipsis';

describe('<Ellipsis />', () => {
  test('renders correctly', () => {
    const component = shallow(<Ellipsis />);
    expect(component).toMatchSnapshot();
  });
});
