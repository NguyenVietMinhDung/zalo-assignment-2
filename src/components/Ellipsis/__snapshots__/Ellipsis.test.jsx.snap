// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Ellipsis /> renders correctly 1`] = `
<span>
  ...
</span>
`;
